project/cmake/addons/CMakeLists.txt

```diff
@@ -413,4 +417,4 @@ endif()
 
 # add custom target "supported_addons" that returns all addons that are supported on this platform
 string(REPLACE ";" " " ALL_ADDONS_BUILDING "${ALL_ADDONS_BUILDING}")
-add_custom_target(supported_addons COMMAND ${CMAKE_COMMAND} -E echo "ALL_ADDONS_BUILDING: ${ALL_ADDONS_BUILDING}" VERBATIM)
\ No newline at end of file
+add_custom_target(supported_addons COMMAND ${CMAKE_COMMAND} -E echo "ALL_ADDONS_BUILDING: ${ALL_ADDONS_BUILDING}" VERBATIM)
```

Comment:
That's the crappy "\ No newline at end of file" diff warning.
First commit probably didn't triggered it because it was a find and replace job from command line.

What me to get rid of it?